api-server/justfile

```diff
@@ -0,0 +1,74 @@
+# load environment vars from .env
+set dotenv-load := true
+
+# display this help
+default:
+  just --list
+
+# build the api-server docker image
+build:
+  docker-compose build api
+
+# publish the api-server docker image to docker.io
+publish:
+  docker login
+  docker-compose push api
```

Comment:
what tag will this get published as. I don't quite get how the repo is being configured

Reply:
I didn't publish the image, but the docker-compose.yml has these directives:
```
  api:
    build: ./
    image: ${IMAGE_NAME}:${IMAGE_TAG:-latest}
```
I.e. the default tag would be "latest".   The simplist way to override this would be something like

```console
$ IMAGE_TAG=development just build
```